### ISSUE TITLE
fix: Change newsletter link to NickyT's newsletter

### ISSUE THREAD

**Issue by BekahHW** [Virtual-Coffee/virtualcoffee.io#1120]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of Change

Typo

### Context for content change

The link we've been putting out in the newsletter for Nick's newsletter is outdated.

### Proposed solution

In some of the the newsletter issues, we link to Nick's newsletter. All the issues can be found in `app/routs/_frontend/newsletter/issues. 

- Replace 24 instances of 
```
		<a href="https://www.getrevue.co/profile/nickytonline/issues/yet-another-newsletter-lol-issue-31-1002343">Nick Taylor's newsletter</a>
```

with 

```
<a href="https://www.nickyt.co/news">Nick Taylor's newsletter </a>
```
The fastest way to do this is to load the project in VS Code, search, and then replace.
![image](https://github.com/Virtual-Coffee/virtualcoffee.io/assets/34313413/98e62e84-6577-4bd3-8d4d-e4213f095f75)

### Code of Conduct

- [X] I've read the Code of Conduct and understand my responsibilities as a member of the Virtual Coffee community

**Comment by nowji** [comment]
Could I be assigned this issue?

**Comment by BekahHW** [comment]
@nowji its all you.

**Comment by nowji** [comment]
I submitted a pull request its just waiting for a review now :)

**Comment by danieltott** [comment]
Closed via #1121
